metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/store/PolarisStoreService.java

```diff
@@ -0,0 +1,41 @@
+package com.netflix.metacat.connector.polaris.store;
+
+import com.netflix.metacat.connector.polaris.store.entities.PolarisDatabaseEntity;
+import com.netflix.metacat.connector.polaris.store.entities.PolarisTableEntity;
+import java.util.List;
+
+/**
+ * Interface methods for Polaris Store CRUD access.
+ */
+public interface PolarisStoreService {
+
+    /**
+     * Creates a database entry.
+     * @param databaseName database name
+     * @return Polaris Database entity.
+     */
+    PolarisDatabaseEntity createDatabase(String databaseName);
```

Comment:
Could this just be the `QualifiedName`?

Reply:
it can't since it doesnt use catalog name..

Reply:
Ah yeah, there are `QualifiedName` APIs to extract out the relevant name parts though. It's the archetype we follow in interfaces in other parts of the codebase and could perhaps be useful from an extensibility standpoint.

Reply:
from a  method signature that takes QN, it is not obvious that we are not using the catalog.
with this signature, we explicitly call out the parts that we need..

Reply:
how about we take this offline so that Jeff is unblocked.

Reply:
Makes sense, either way should be fine. Just that this breakdown should happen somewhere in the stack and was thinking it'd be rather useful to do it at the lowest layer. Say tomorrow we might want to use some of the QN methods at this layer too or start logging it (we typically debug by searching for "prodhive/foo/bar") or storing it in a separate layer etc.